Resources/Prototypes/_RMC14/Entities/Objects/Misc/floodlights.yml

```diff
@@ -0,0 +1,187 @@
+- type: entity
+  parent: BaseStructureDynamic
+  id: RMCSmallFloodlight
+  name: small floodlight
+  description: A small bit of light in the darkness.
+  components:
+  - type: Sprite
+    noRot: true
+    sprite: _RMC14/Objects/Misc/Lights/floodlight.rsi
+    layers:
+      - state: smallfloodon # Temporary until power works.
+      - state: smallflood
+        visible: false
+        map: [ "light" ]
+  - type: InteractionOutline
+  - type: Repairable
+  - type: Damageable
+    damageContainer: Inorganic
+    damageModifierSet: Metallic
+  - type: PointLight
+    enabled: true
+    radius: 6
+    netsync: false
+    energy: 5
+    offset: "0, 0"
+    color: "#ECFFFF"
+    mask: /Textures/_RMC14/Effects/LightMasks/quad_cone.png
+  - type: Destructible
+    thresholds:
+    - trigger:
+        !type:DamageTrigger
+        damage: 150
```

Comment:
I don't think these are destructible in 13 but I'd have to check

Reply:
I wanted to make sure they're not used as indestructible barricades

Reply:
Makes sense